concepticondata/conceptlists/Teit-1908-693.tsv

```diff
@@ -0,0 +1,694 @@
+Teit-1908-693-216	216	192	soup	1547	SOUP	56
+Teit-1908-693-217	217	193	pipe	1124	PIPE	56
+Teit-1908-693-218	218	194	pipe-stem			56
+Teit-1908-693-219	219	195	rope	1218	ROPE	56
+Teit-1908-693-220	220		hide rope			56
+Teit-1908-693-221	221	196	Indian hemp	2041	HEMP	56
+Teit-1908-693-222	222	197	drum	908	DRUM	56
+Teit-1908-693-223	223	198	drumstick			56
+Teit-1908-693-224	224	199	root-digger			56
+Teit-1908-693-225	225	200	[root-digger] handle			56
+Teit-1908-693-226	226	201	tump-line	294	TUMPLINE	56
+Teit-1908-693-227	227	202	bark basket			58
+Teit-1908-693-228	228	203	woven basket			58
+Teit-1908-693-229	229	204	woven bag			58
+Teit-1908-693-230	230	205	doll	1659	DOLL	58
+Teit-1908-693-231	231	206	stilts	2593	STILTS	58
+Teit-1908-693-232	232	207	swing	3101	SWING	58
+Teit-1908-693-233	233	208	red paint			58
+Teit-1908-693-234	234	209	yellow paint			58
+Teit-1908-693-235	235	210	tattoo marks	68	TATTOO	58
+Teit-1908-693-236	236	211	song	1096	SONG	58
+Teit-1908-693-237	237	212	name	1405	NAME	58
+Teit-1908-693-238	238	213	tent (any kind)	1316	TENT	58
+Teit-1908-693-239	239		tent of whites			58
+Teit-1908-693-240	240		conical lodge			58
+Teit-1908-693-241	241	214	house	1252	HOUSE	58
+Teit-1908-693-242	242	215	town	1391	TOWN	58
+Teit-1908-693-243	243	216	earth lodge			58
+Teit-1908-693-244	244	217	mat lodge			58
+Teit-1908-693-245	245		conical			58
+Teit-1908-693-246	246	218	bark lodge			58
+Teit-1908-693-247	247	219	skin lodge			58
+Teit-1908-693-248	248	220	door (in general)	1567	DOOR	58
+Teit-1908-693-249	249	221	doorflap			60
+Teit-1908-693-250	250	222	room	1450	ROOM	60
+Teit-1908-693-251	251	223	bed	1663	BED	60
+Teit-1908-693-252	252	224	bath			60
+Teit-1908-693-253	253	226	sweathouse			60
+Teit-1908-693-254	254	227	hotwater house			60
+Teit-1908-693-255	255	228	hot water bath			60
+Teit-1908-693-256	256	229	fire	221	FIRE	60
+Teit-1908-693-257	257	230	ashes	646	ASH	60
+Teit-1908-693-258	258	231	smoke	778	SMOKE (EXHAUST)	60
+Teit-1908-693-259	259	232	snoot	1221	NOSE	60
+Teit-1908-693-260	260	233	firewood	10	FIREWOOD	60
+Teit-1908-693-261	261	234	charcoal	1188	CHARCOAL	60
+Teit-1908-693-262	262	235	wrist guard			60
+Teit-1908-693-263	263	236	bow (in general)	994	BOW	60
+Teit-1908-693-264	264	237	bow-string	2119	BOW STRING	60
+Teit-1908-693-265	265	238	arrow	977	ARROW	60
+Teit-1908-693-266	266	239	arrow head	3255	POINT (POINTED END)	60
+Teit-1908-693-267	267	240	iron [head]			60
+Teit-1908-693-268	268	241	arrow feathers			60
+Teit-1908-693-269	269	242	quiver	995	QUIVER (FOR ARROWS)	60
+Teit-1908-693-270	270	243	shield	1064	SHIELD	60
+Teit-1908-693-271	271	244	war club	1763	CLUB	60
+Teit-1908-693-272	272		stone head in skin			60
+Teit-1908-693-273	273		stone head			60
+Teit-1908-693-274	274		stoneheaded club			60
+Teit-1908-693-275	275		stone head movable			60
+Teit-1908-693-276	276	245	war spear	945	SPEAR	60
+Teit-1908-693-277	277	246	fish spear two prong			62
+Teit-1908-693-278	278		handle [of fish spear]	1254	HANDLE	62
```

Comment:
I'd unmap since there are other types of "handles" in the list and they seem to be specific.